.github/workflows/main.yml

```diff
@@ -239,6 +239,22 @@ jobs:
   #       WBINDGEN_I_PROMISE_JS_SYNTAX_WORKS_IN_NODE: 1
   #   - run: cargo build --manifest-path crates/web-sys/Cargo.toml --target wasm32-unknown-unknown --features "Node Window Document"
 
+  # This checks that the output of the CLI is actually valid JavaScript and TypeScript
+  test_cli_reference_typescript:
+    name: Run CLI reference TypeScript check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: '20'
```

Comment:
What is your thinking there?

I would either use the default one or preferably just the latest version.

Reply:
I just copied the code from another config and 20 seemed like a fine version.

But we can go to latest LTS. I don't want to use the actual latest release, because nodejs has had multiple versions in the past that flat out broke certain npm packages, and I don't want to deal with that here.

Reply:
Sounds fine.